src/js/view/controls/next-display-icon.js

```diff
@@ -0,0 +1,30 @@
+define([
+    'utils/helpers',
+    'utils/ui',
+    'templates/display-icon.html'
```

Comment:
It would be great if we could somehow remove template imports so we don't need the handlebars-loader when running unit tests